pkg/templates/rancherd-22-addons.yaml

```diff
@@ -375,12 +375,18 @@ resources:
           pullPolicy: IfNotPresent
         deschedulingInterval: 5m
         replicas: 1
+        cmdOptions:
+          v: 3
+          feature-gates: EvictionsInBackground=true
         deschedulerPolicy:
+          maxNoOfPodsToEvictPerNode: 10
```

Comment:
10 Pods might be too many?
Maybe 5 should be enough... consist with the Longhorn rebuild concurrency number.

WDYT?
cc @ibrokethecloud

Reply:
sounds good.. we can document it, and user can always tweak it if needed

Reply:
we may need to make sure when we update the addon during upgrades not to replace this

Reply:
Document PR: https://github.com/harvester/docs/pull/920

> we may need to make sure when we update the addon during upgrades not to replace this

Yes, we only change `enabled` field.